src/js/components/header.tsx

```diff
@@ -1,27 +1,90 @@
+import Dropdown from '@salesforce/design-system-react/components/menu-dropdown';
 import PageHeader from '@salesforce/design-system-react/components/page-header';
 import PageHeaderControl from '@salesforce/design-system-react/components/page-header/control';
-import React from 'react';
+import i18n from 'i18next';
+import React, { useCallback, useEffect, useState } from 'react';
+import { CallBackProps, STATUS } from 'react-joyride';
 import { useSelector } from 'react-redux';
 import { Link } from 'react-router-dom';
 
+import backpackSvg from '!raw-loader!~img/backpack-sm.svg';
+import mapSvg from '!raw-loader!~img/map-sm.svg';
+import seesawSvg from '!raw-loader!~img/seesaw-sm.svg';
 import Errors from '~js/components/apiErrors';
 import OfflineAlert from '~js/components/offlineAlert';
 import Toasts from '~js/components/toasts';
+import { TourType } from '~js/components/tour/landing';
+import PlanTour from '~js/components/tour/plan';
 import UserInfo from '~js/components/user/info';
 import { selectSocketState } from '~js/store/socket/selectors';
 import { selectUserState } from '~js/store/user/selectors';
-import routes from '~js/utils/routes';
+import routes, { routePatterns } from '~js/utils/routes';
 
-const Header = () => {
+const Header = ({ page }: { page: string }) => {
   const user = useSelector(selectUserState);
   const socket = useSelector(selectSocketState);
-
+  const [tourRunning, setTourRunning] = useState<TourType | null>(null);
+  const [hideTourHelper, setHideTourHelper] = useState(
+    page === routePatterns.project_list(),
+  );
+  const doRunTour = (type: TourType) => setTourRunning(type);
+  const handleTourCallback = useCallback((data: CallBackProps) => {
+    const { status } = data;
+    const finishedStatuses: string[] = [STATUS.FINISHED, STATUS.SKIPPED];
+    if (finishedStatuses.includes(status)) {
+      setTourRunning(null);
+    }
+  }, []);
+  const doShowHelper = !hideTourHelper && user?.onboarded_at;
+  useEffect(() => {
+    setHideTourHelper(page === routePatterns.project_list());
+  }, [page]);
   const controls = () => (
     <PageHeaderControl className="slds-grid slds-grid_vertical-align-center">
+      {doShowHelper && (
+        <div className="slds-col_padded">
+          <Dropdown
```

Comment:
@jgerigmeyer also meant to mention, these icons aren't showing up; passing with `category` may not be needed, but makes we wonder if we do need to use the [listItemRenderer](https://react.lightningdesignsystem.com/components/menu-dropdowns/#prop-listItemRenderer) prop instead?